backend/.env

```diff
@@ -0,0 +1 @@
+ATLAS_URI=mmongodb+srv://kuehma02:project-2020@cluster0-o9rln.azure.mongodb.net/test?retryWrites=true&w=majority
```

Comment:
SHOOT THIS SHOULD NOT END UP HERE. We might have to add another line in the .gitignore. When i did this before I ended up using this link: http://www.codeblocq.com/2016/01/Untrack-files-already-added-to-git-repository-based-on-gitignore/

Reply:
Review this

Reply:
Okay so should that be removed before we merge this in?

Reply:
Like is it a security concern to expose our database endpoint?

Reply:
I think we can merge it in and then remove it, but I'm not sure how to take it out without just tossing the entire commit.

Reply:
But yeah we don't want this sitting in our repo

Reply:
If you add it to the gitignore, run those commands in the link you sent, and then push to your branch again, it should update the PR!